conf/appNICs.routes

```diff
@@ -1,30 +1,34 @@
+POST        /:taxYear/national-insurance-contributions/class-4-exemption-reasons/diver-diving-supervisor                             controllers.journeys.nics.Class4DivingExemptController.onSubmit(taxYear: TaxYear, mode: Mode = NormalMode)
+GET         /:taxYear/national-insurance-contributions/change-class-4-exemption-reasons/diver-diving-supervisor                      controllers.journeys.nics.Class4DivingExemptController.onPageLoad(taxYear: TaxYear, mode: Mode = CheckMode)
+POST        /:taxYear/national-insurance-contributions/change-class-4-exemption-reasons/diver-diving-supervisor                      controllers.journeys.nics.Class4DivingExemptController.onSubmit(taxYear: TaxYear, mode: Mode = CheckMode)
 
-GET         /:taxYear/national-insurance-contributions/class-4-exemption-reasons/trustee-executor-administrator               controllers.journeys.nics.Class4NonDivingExemptController.onPageLoad(taxYear: TaxYear, mode: Mode = NormalMode)
-POST        /:taxYear/national-insurance-contributions/class-4-exemption-reasons/trustee-executor-administrator               controllers.journeys.nics.Class4NonDivingExemptController.onSubmit(taxYear: TaxYear, mode: Mode = NormalMode)
-GET         /:taxYear/national-insurance-contributions/change-class-4-exemption-reasons/trustee-executor-administrator        controllers.journeys.nics.Class4NonDivingExemptController.onPageLoad(taxYear: TaxYear, mode: Mode = CheckMode)
-POST        /:taxYear/national-insurance-contributions/change-class-4-exemption-reasons/trustee-executor-administrator        controllers.journeys.nics.Class4NonDivingExemptController.onSubmit(taxYear: TaxYear, mode: Mode = CheckMode)
+GET         /:taxYear/national-insurance-contributions/class-4-exemption-reasons/trustee-executor-administrator                      controllers.journeys.nics.Class4NonDivingExemptController.onPageLoad(taxYear: TaxYear, mode: Mode = NormalMode)
+POST        /:taxYear/national-insurance-contributions/class-4-exemption-reasons/trustee-executor-administrator                      controllers.journeys.nics.Class4NonDivingExemptController.onSubmit(taxYear: TaxYear, mode: Mode = NormalMode)
+GET         /:taxYear/national-insurance-contributions/change-class-4-exemption-reasons/trustee-executor-administrator               controllers.journeys.nics.Class4NonDivingExemptController.onPageLoad(taxYear: TaxYear, mode: Mode = CheckMode)
+POST        /:taxYear/national-insurance-contributions/change-class-4-exemption-reasons/trustee-executor-administrator               controllers.journeys.nics.Class4NonDivingExemptController.onSubmit(taxYear: TaxYear, mode: Mode = CheckMode)
+GET         /:taxYear/national-insurance-contributions/class-4-exemption-reasons/trustee-executor-administrator-single               controllers.journeys.nics.Class4NonDivingExemptSingleBusinessController.onPageLoad(taxYear: TaxYear, mode: Mode = NormalMode)
```

Comment:
Place can we put a space to separate the different controllers please

Reply:
Done